reledpar.dtx

```diff
@@ -1457,14 +1457,14 @@
 % \begin{macro}{\led@error@edtext@later@now}
 %    \begin{macrocode}
 \newcommand{\led@error@edtext@later@now}{%
-  \reledpar@error{You call \the\edtext@later\space \string\edtextlater\space commands but  \the\edtext@now\space \string\edtextnow\space commands}{\@ehc}%
+  \reledpar@error{You called the \the\edtext@later\space \string\edtextlater\space commands, not the \the\edtext@now\space \string\edtextnow\space commands}{\@ehc}% (did I follow your intent correctly here?)
```

Comment:
Understood and a helpful grammatical tip about how you write your error messages. I won’t forget that.

M


> On 2016-07-01, at 03:49 , Maïeul Rouquette <notifications@github.com> wrote:
> 
> In reledpar.dtx <https://github.com/maieul/ledmac/pull/550#discussion_r69261835>:
> 
> > @@ -1457,14 +1457,14 @@
> >  % \begin{macro}{\led@error@edtext@later@now}
> >  %    \begin{macrocode}
> >  \newcommand{\led@error@edtext@later@now}{%
> > -  \reledpar@error{You call \the\edtext@later\space \string\edtextlater\space commands but  \the\edtext@now\space \string\edtextnow\space commands}{\@ehc}%
> > +  \reledpar@error{You called the \the\edtext@later\space \string\edtextlater\space commands, not the \the\edtext@now\space \string\edtextnow\space commands}{\@ehc}% (did I follow your intent correctly here?)
> now,1) all the message of error are in present, not preterit 2) this specific message compare two number, so the "but" is better than the "not the".
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/maieul/ledmac/pull/550/files/abc94f75401202c18ccac19e1d798f85234d748f#r69261835>, or mute the thread <https://github.com/notifications/unsubscribe/AHG-QGRvLmpyVKfMtAvNzNjhe8lcxQJwks5qRMabgaJpZM4JC5Lv>.
>